br/pkg/lightning/restore/table_restore.go

```diff
@@ -850,8 +850,16 @@ func (tr *TableRestore) importKV(
 	engineID int32,
 ) error {
 	task := closedEngine.Logger().Begin(zap.InfoLevel, "import and cleanup engine")
-
-	err := closedEngine.Import(ctx)
+	regionSplitSize := int64(rc.cfg.TikvImporter.RegionSplitSize)
+	if rc.cfg.TikvImporter.RegionSplitSize == config.SplitRegionSize && rc.taskMgr != nil {
```

Comment:
🤔 

this makes `'96 MiB'` in the config file a very magic number. 

i suggest either changing the config's default `region-split-size` to 0 or -1, or entirely deprecate the setting.